yarn.lock

```diff
@@ -1806,6 +1815,13 @@
     "@elastic/apm-rum" "^5.9.1"
     hoist-non-react-statics "^3.3.0"
 
+"@elastic/apm-rum@^5.10.0":
```

Comment:
I would consolidate this entry with the one below as the two semver's overlap

Reply:
I pushed I pushed https://github.com/elastic/kibana/pull/162722/commits/4ff1763ff93fe9655320844419d722f12470f5c1